app.py

```diff
@@ -40,6 +40,18 @@ def mirror(name):
 
 # TODO: Implement the rest of the API here!
 
+@app.route('/users')
+def users():
+	return create_response(db.get('users'))
```

Comment:
You should pass a dictionary of users into `create_response` (check the docstring at the beginning of it), e.g. `return create_response({'users': db.get('users')})`